src/components/App.js

```diff
@@ -1,15 +1,33 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-import calculate from '../logic/calculate'; /*eslint-disable-line*/
-
-function App() {
-  return (
-    <div id="root-app">
-      <Display />
-      <ButtonPanel />
-    </div>
-  );
+import calculate from '../logic/calculate';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      total: null,
+      next: null,
+      operation: null, /*eslint-disable-line*/
```

Comment:
we are indirectly changing the rules by disabling this rule, let's fix the codebase instead.